### PR TITLE
Feature/posts (Posts, navbar issue fix and logout)

### PR DESCRIPTION
**Pull request by mariiadevphc**
- Post Feed (`/posts`)
- Create Post form to add post to Post Feed (`/posts/create`)
- Fixed Navbar issue (navbar didn't appear after login, had to refresh)
- Logout - click on your name in navbar for menu to appear, and Logout will be part of the options (`/logout`)

### CODE REVIEW

File: frontend/src/pages/Orientation/Pages/RelativeLocation.tsx [hunk 1]
```diff
@@ -128,18 +128,24 @@ function RelativeLocation(props: any) {
 				needs.
 			</p>
 			<div className="location-container">
+				<img src="/icons/location.png" className="location-icon" />
```
Comment (nick-cd) [inline frontend/src/pages/Orientation/Pages/RelativeLocation.tsx:131]:
```suggestion
				<img src="/icons/location.png" alt="" className="location-icon" />
```

File: frontend/src/pages/Orientation/Pages/Interests.tsx [hunk 2]
```diff
@@ -16,16 +16,16 @@ function Interests(props: any) {
 			<h3 className="main-heading">Let’s get you stitched in.</h3>
 			<p className="sub-heading">What are your interests?</p>
 			<div className="search-container">
+				<img
+					src="https://cdn1.iconfinder.com/data/icons/hawcons/32/698627-icon-111-search-512.png"
```
Comment (nick-cd) [inline frontend/src/pages/Orientation/Pages/Interests.tsx:20]:
```suggestion
					src="https://cdn1.iconfinder.com/data/icons/hawcons/32/698627-icon-111-search-512.png"
alt=""
```

**Review by nick-cd — CHANGES_REQUESTED** [review]
Thank you for your hard work here!

Can you quickly add an empty `alt` atribute to your decorative images? Once you do, I'll merge!

File: frontend/src/pages/Orientation/Pages/Interests.tsx (hunk 2 above)
Comment (mariiadevphc) [reply frontend/src/pages/Orientation/Pages/Interests.tsx:20]: fixed with 61fc7db1aa0500c49dfe22fc232b880df2fcbb01

File: frontend/src/pages/Orientation/Pages/RelativeLocation.tsx (hunk 1 above)
Comment (mariiadevphc) [reply frontend/src/pages/Orientation/Pages/RelativeLocation.tsx]: fixed with 61fc7db1aa0500c49dfe22fc232b880df2fcbb01

**Review by nick-cd — APPROVED** [review]

**Pull request merged**